docs/standard/analyzers/platform-compat-analyzer.md

```diff
@@ -0,0 +1,363 @@
+
+But using platform-dependent APIs on a component makes the code no longer work across all platforms. We needed a way to detect this at design-time so that developers get diagnostics when they inadvertently use platform-specific APIs. To achieve this goal we introduced [platform compatibility analyzer](/visualstudio/code-quality/ca1416) and complementary APIs help developers identify and use platform-specific APIs where appropriate, the new APIs include:
+
+- Added <xref:System.Runtime.Versioning.SupportedOSPlatformAttribute> to annotate APIs as being platform-specific and <xref:System.Runtime.Versioning.UnsupportedOSPlatformAttribute> to express that and API is unsupported on a particular OS, optionally including the version numbers. These attributes have been applied to platform-specific APIs in the core .NET libraries (still in progress).
+- Added `Is<Platform>()` and `Is<Platform>VersionAtLeast(int major, int minor = 0, int build = 0, int revision = 0)` style static methods in the <xref:System.OperatingSystem?displayProperty=nameWithType> class for safely calling platform dependent APIs, for example, <xref:System.OperatingSystem.IsWindows?displayProperty=nameWithType> could be used for guarding a call of windows specific APIs and  <xref:System.OperatingSystem.IsWindowsVersionAtLeast%2A?displayProperty=nameWithType>() could be used for guarding a versioned windows specific API call. Learn [more](#the-platform-specific-api-calls-in-the-above-examples-guarded-with-guard-methods) about how they used as guards of platform-specific API references.
+
+> [!Tip]
+> Platform compatibility analyzer upgrades/replaces [Discovering cross-platform issues](../../standard/analyzers/api-analyzer.md#discover-cross-platform-issues) feature of the [.NET API analyzer](../../standard/analyzers/api-analyzer.md).
+
+## Prerequisites
+
+Platform compatibility analyzer is one of the Roslyn code quality analyzers. Starting from .NET 5.0, these analyzers are [included with the .NET SDK](../../fundamentals/productivity/code-analysis.md). For older .Net versions you can install code quality analyzers from [NuGet package](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers). The platform compatibility analyzer is enabled by default only for projects targeting `net5.0` or later. However, you can [enable](/visualstudio/code-quality/ca1416.md#configurability) it for projects targeting other frameworks.
```

Comment:
```suggestion
The platform compatibility analyzer is one of the Roslyn code quality analyzers. Starting in .NET 5.0, these analyzers are [included with the .NET SDK](../../fundamentals/productivity/code-analysis.md). For older .NET versions, you can install code quality analyzers from [NuGet package](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers). The platform compatibility analyzer is enabled by default only for projects that target `net5.0` or a later version. However, you can [enable](/visualstudio/code-quality/ca1416.md#configurability) it for projects that target other frameworks.
```

Does the NuGet package include the platform compatibility analyzer?

Reply:
I don't see NuGet package for 5.0, older versions wouldn't have it, so maybe better remove this `NuGet package` section